web/core/components/issues/workspace-draft/draft-issue-block.tsx

```
@@ -0,0 +1,127 @@
+export const DraftIssueBlock: FC<Props> = observer((props) => {
+  // props
+  const { workspaceSlug, issueId } = props;
+  // hooks
+  const { getIssueById, updateIssue, deleteIssue, moveIssue } = useWorkspaceDraftIssues();
+  const { sidebarCollapsed: isSidebarCollapsed } = useAppTheme();
+  const { getProjectIdentifierById } = useProject();
+  // ref
+  const issueRef = useRef<HTMLDivElement | null>(null);
+  // derived values
+  const issue = getIssueById(issueId);
+  const projectIdentifier = (issue && issue.project_id && getProjectIdentifierById(issue.project_id)) || undefined;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)